x-pack/plugins/enterprise_search/public/applications/app_search/icons/engine_icon.tsx

```diff
@@ -7,13 +7,14 @@
 
 import React from 'react';
 
-export const EngineIcon: React.FC = () => (
+export const EngineIcon: React.FC = ({ ...props }) => (
   <svg
     xmlns="http://www.w3.org/2000/svg"
+    className="euiIcon"
+    width={18}
+    height={18}
+    {...props}
     viewBox="0 0 18 18"
-    width="18"
-    height="18"
-    className="euiIcon engineIcon"
```

Comment:
removing the `engineIcon` causes the icon to no longer be center aligned via CSS:

<img width="141" alt="Screen Shot 2021-04-22 at 12 05 21 PM" src="https://user-images.githubusercontent.com/549407/115771641-0e2f4200-a363-11eb-91b9-89d0e9993d17.png">

we need to add an EuiFlexWrapper then if we don't want this, and we should also delete the CSS for `.engineIcon`